lib/protobuf/field/base_field.rb

```diff
@@ -234,6 +235,10 @@ def define_getter
         ::Protobuf.field_deprecator.deprecate_method(message_class, method_name) if field.deprecated?
       end
 
+      def define_decode_setter
+        # empty for now
```

Comment:
Only a few variants of this method. Let's bring the duplicated common implementation here.